app/views/employees/_list.html.erb

```diff
@@ -35,6 +38,17 @@
             <%= employee.administrator_id %>
           </td>
           <td>
-            <% end %>
+            <%= link_to "Show", employee_path(employee), class: "btn btn-info" %>
+          </td>
+          <td>
+            <%= link_to "Edit", edit_employee_path(employee), class: "btn btn-warning" %>
+          </td>
+          
+          <td>
+            <%= link_to "Delete", employee_path(employee) ,class: "btn btn-danger" %>
```

Comment:
Missing space after comma
but everyhting looks good to me!
`<%= link_to "Delete", employee_path(employee), class: "btn btn-danger" %>`

Reply:
ok, i will correct that little mistake